Bio/Align/AlignInfo.py

```diff
@@ -178,6 +179,144 @@ def gap_consensus(self, threshold=.7, ambiguous="X",
 
         return Seq(consensus, consensus_alpha)
 
+    def iupac_consensus(self, ambiguous="X", 
+                        consensus_alpha=None, require_multiple=0):
+        """Output a fast consensus sequence of the DNA alignment.
+
+        This doesn't do anything fancy at all. It will just go through the
+        sequence residue by residue(ie. A or G or T or C for DNA) 
+        in all sequences in the alignment. It will add IUPAC nucleotide 
+        ambiguity bases where appliciable. This does not account for number
+        of bases, if only 1 base differs in an alignment of 100, 
+        This will will call an ambiguity. A very conservative approach. 
+
+        Arguments:
+         - ambiguous - The ambiguous character to be added when the threshold is
+           not reached.
+         - consensus_alpha - The alphabet to return for the consensus sequence.
+           If this is None, then we will try to guess the alphabet.
+         - require_multiple - If set as 1, this will require that more than
```

Comment:
It looks like ``require_multiple`` is used as a boolean (true/false), in which case using the magic value ``1`` is unclear.

Reply:
I didn't write the original code that included this boolean. I've left it alone because I was not 100% on what 'require_multiple' is doing here. Do you have a suggestion on how to fix this? If so, we will need to look at the original published `dumb_consensus` and `gap_consensus` functions, and fix those as well.

Reply:
The original code probably predates Python having True/False - bits of Biopython are really old. We could as part of your pull request update the old functions to use boolean default instead for consistency? But as you realise, any change can have side effects.